Changes

```diff
@@ -1,5 +1,43 @@
 Release history for Zonemaster component Zonemaster-Backend
 
+v8.0.0 2021-12-03 (public release version)
+
+ [Features]
+ - Adds Finnish translation of error messages (#921, #911)
+ - Replaces CentOS with Rocky Linux (#908, #895)
+ - Adds upgrade documentation (#907, #879, #618)
+ - Adds Swedish translation of error messages (#899)
+ - Adds translation of error messages (#891)
+ - Drops support for PostgreSQL version 9 and before (#892)
+ - Adds support for collecting metrics (#844)
+ - Optimization of batches when PostgreSQL is used (#890)
+ - Improved result history response in API (#837, #830)
+ - Deprecates country code (#796, #794)
+ - Makes database port configurable (#755, #496)
```

Comment:
Dropping support for something isn't a feature - it's a breaking change. E.g. PostgreSQL. Or do we have reason to believe that PostgreSQL will still work, but we just haven't ensured that in release testing? In that case we should probably find a different wording.

Maybe group these so that similar items appear next to each other? E.g. updates related to translations.

Reply:
At some time we decided to change the major version, which means that we think it is a breaking change, but I do not think that is because of the PostgreSQL. Do you remember?

Reply:
I have created a section for breaking changes, and done some grouping. The new code will not work with old PostgreSQL.

Reply:
Yeah, #892 says we've removed a workaround for PostgreSQL < 10.